docker-compose.test.yml

```diff
@@ -0,0 +1,6 @@
+version: '3.8'
```

Comment:
Chose to keep this as a docker compose even though it's technically redundant bc it makes the rebuild interface a bit cleaner